raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -48,8 +48,8 @@ def __init__(
             self.dashboard_input.cohortData = []
 
         self._feature_length = len(self.dashboard_input.dataset.feature_names)
-        if hasattr(analysis, ManagerNames.ERROR_ANALYSIS):
```

Comment:
why is this part commented out? I think we should still add it back in?

Reply:
fixed, that was just for testing :)